api/src/auth_routes.ts

```diff
@@ -28,6 +28,19 @@ import {registerLocalUser} from './auth_providers/local';
 import {body, validationResult} from 'express-validator';
 import {getInvite} from './couchdb/invites';
 import {acceptInvite} from './registration';
+import {generateUserToken} from './authkeys/create';
+import {NextFunction, Request, Response, Router} from 'express';
+
+interface RequestQueryRedirect {
+  redirect: string;
+}
+interface PostRegisterRequestBody {
+  username: string;
+  password: string;
+  email: string;
+  repeat: string;
+  name: string;
+}
 
```

Comment:
If these are returns/inputs to API - should go in data model package with the other api.ts interfaces?